packages/office-addin-manifest/src/manifestHandler/getManifestHandler.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { ManifestHandler } from "./manifestHandler";
+import { ManifestHandlerJson } from "./manifestHandlerJson";
+import { ManifestHandlerXml } from "./manifestHandlerXml";
+
+function isJsonFile(path: string) {
+  return path.endsWith(".json");
+}
+
+export function getManifestHandler(manifestPath: string): ManifestHandler {
+  if (isJsonFile(manifestPath)) {
```

Comment:
I don't want to add the XML check now because I don't want to change the current behavior of the code.
Right now, any file goes to the XML handler and I want to keep it that way to avoid messing too much with production code for now.

Reply:
So what happens currently when the wrong file is sent?  How do you plan on making sure this is done later if not now?

Reply:
If a wrong file is sent, the same thing that happened previously will continue happening:
The following error will be thrown:
```reject(new Error(`Unable to parse the manifest file: ${manifestPath}. \n${parseError}`));```
I don't think we need to care about it too much. The `ManifestHandlerXML` already handles all kind of files (by throwing an error).

Reply:
It's always good to think about having good, informative, and clear errors. Nothing is more frustrating that the user having to guess what is wrong. If you can tell the difference between a parse error or another condition, it's good to have a more specific message. If the parse error can be clear about the problem, and show that a non-XML file cannot be parsed propertly, then that can be sufficient.

Reply:
I agree @akrantz. I just added a more clarifying message in case an unsupported extension is given to this method.